### ISSUE TITLE
Cannot add elements to an empty upgraded element during hydration

### ISSUE THREAD

**Issue by lawnsea** [ampproject/worker-dom#10]
If the worker script for an empty upgraded node adds elements before it runs to completion, the changes are not replicated to the parent page. If the worker script adds the elements on a later turn of the event loop, the changes are replicated.

As a user, I would expect to be able to add content to the upgraded element during the initial execution of the worker script.

Repro: https://glitch.com/edit/#!/spot-flame

**Comment by kristoferbaxter** [comment]
Thank you for the report and especially the repro case.

I'm going to create a larger issue to track an improvement for the test suite to allow us to properly test this type of code so regressions do not occur.

**Comment by gravityrail** [comment]
I got bitten by this same bug - spent hours on it! Definitely looks like the hydrate method is receiving the correct DOM structure from the initial pass through the code, but somehow it's not being rendered to the upgraded node in the parent page. 

Note that if you render the same content twice, e.g. `<h1>Hello World</h1>`, it will still not mutate the page. The content needs to vary between the hydrate call and the mutate call.

This example worker code:

```
function createExampleNode( titleText ) {
	const h1 = document.createElement('h1');
	const title = document.createTextNode(titleText);
	h1.appendChild( title )
	document.body.appendChild( h1 );
}
createExampleNode('hello world on worker boot');
setTimeout(() => {
	createExampleNode('hello world from setTimeout');
}, 100);
```

Adds the `hello world from setTimeout` node but not the `hello world on worker boot` node.

**Comment by kristoferbaxter** [comment]
This was fixed with release `0.2.0`! Apologies for the delays.
